src/lib/Hydra/Schema/Buildsbymaintainer.pm

```diff
@@ -0,0 +1,111 @@
+use utf8;
+package Hydra::Schema::Buildsbymaintainer;
```

Comment:
Can you add `"buildsbymaintainer" => "BuildsByMaintainer",` to `src/sql/updated-dbix.pl`'s moniker_map?

Reply:
@grahamc fixed :)